src/reducers/app/thunks.ts

```diff
@@ -47,10 +47,33 @@ const getAttributeNames = (attributes) => {
 };
 
 const getInitialColumns = (entity, columns) => {
-  if (entity !== 'blocks') {
-    const newColumns = columns.slice(0, 6);
-    return newColumns;
-  } else {
+  const sorted = columns.sort((a, b) =>
+    a.displayName.localeCompare(b.displayName)
+  );
+
+  if (entity === 'operations') {
+    let operationColumns: string[] = [];
+
+    operationColumns.push(sorted.filter(c => c.name === 'timestamp')[0]);
```

Comment:
Please use an filter

Reply:
not sure where this code came from, investigating now.

Reply:
We spoke about using a filter function yesterday, but I haven't found a way with filter to return them in the particular order that we want, I pushed up the changes using the reduce function but if you have a better idea using filter I'm all ears.